symengine/printers/strprinter.h

```diff
@@ -190,10 +190,12 @@ class StrPrinter : public BaseVisitor<StrPrinter>
     void bvisit(const ComplexMPC &x);
 #endif
     void bvisit(const NumberWrapper &x);
+    void bvisit(const DenseMatrix &x);
 
     std::string apply(const RCP<const Basic> &b);
     std::string apply(const vec_basic &v);
     std::string apply(const Basic &b);
+    std::string apply(const DenseMatrix &b);
```

Comment:
Is this needed?

Reply:
Not needed for the latex printing indeed, but I could not find printing to a plain string for `DenseMatrix`. With this PR, the following is allowed:
```
    DenseMatrix d = DenseMatrix(3,2);
    StrPrinter strPrinter;
    std::cout << strPrinter.apply(d);
```
or just `str(d)`.